locales/fr.js

```diff
@@ -168,4 +168,19 @@ export default {
   //Phase Banner
   Alpha: "Alpha",
   alphaText: "Ce site changera au fur et à mesure que nous testons des idées.",
+
+  //Province options
+  alberta: "L’Alberta",
```

Comment:
minor pick usually there is no article in french prov names for a select

Reply:
This was actually one thing I was unsure about. I included the French translation to stay consistent but if it isn't required I could just remove the translation. This would also solve the your other comment I believe.